.gitattributes

```diff
@@ -1 +1,2 @@
+* text=auto
```

Comment:
do you know what this do?

Reply:
From the docs for the text setting:

`This attribute enables and controls end-of-line normalization. When a text file is normalized, its line endings are converted to LF in the repository.`

And auto means that 

`the path is marked for automatic end-of-line conversion. If Git decides that the content is text, its line endings are converted to LF on checkin.`

So whenever you commit a text file, the line endings are converted to LF no matter if you're on Windows or Linux.